tests/integration_tests/test_game.py

```diff
@@ -272,3 +272,55 @@ async def test_partial_game_ended_rates_game(lobby_server, tmp_user):
     # The game should only be rated once
     with pytest.raises(asyncio.TimeoutError):
         await read_until_command(host_proto, "player_info", timeout=10)
+
+
+@fast_forward(5)
+async def test_gamestate_ended_clears_references(
```

Comment:
btw: I have practice at work, where I'm describing the problem that you're trying to test.
I know, that test naming not every time can be descriptive.
Like it's here: https://github.com/norraxx/faf-aio-replayserver/blob/master/tests/server/unitttests/test_replay.py#L47